CONTRIBUTING.md

```diff
@@ -144,6 +141,75 @@ If you do not have any changes on Theia or Che-theia extension, you could just b
 and run these plugins with the existing che-theia app:
 `run ... HOSTED che-theia + container-plugin` or `run ... HOSTED che-theia + workspace-plugin`
 
+## Che-Theia development on che.openshift.io
+
+There is a [devfile](https://github.com/eclipse/che-theia/blob/osio-devfile/devfiles/hosted-che-dogfooding.devfile.yaml) to develop Che-Theia on [che.openshift.io](https://che.openshift.io)
+
+Projects directory which is a Persistent Volume on [che.openshift.io](https://che.openshift.io) is limited to 1 gigabyte. It's not enough to build Che-Theia.
+The main idea is to use `/tmp/theia` directotry. Amount for this directory is limited to 3 gigabytes, which is enough to build Che-Theia and run.
+In comparing with `/projects`, temporary directory is not persisted and is cleared on workspace stop.
+
+### Create workspace
+
+Use the badge to create a workspace using factory
+
+[![Try it on che.openshift.io](https://img.shields.io/static/v1?label=che&message=openshift.io&color=orange)](https://che.openshift.io/f?url=https://raw.githubusercontent.com/eclipse/che-theia/osio-devfile/devfiles/hosted-che-dogfooding.devfile.yaml)
+
+### Step 1: Copy sources to `/tmp/theia` directory
+
+To copy sources to `/tmp/theia` directory use `'1. Rsync Theia'` command. With rsync it's become possible to edit sources in `/projects` with the following synchronizing with `/tmp/theia`.
+
+```
+[che-dev]
+$ rsync -rtv --exclude='node_mobules' /projects/theia/ /tmp/theia/
+```
+
+### Step 2: Initlalize Che-Theia
+
+To initialize Che-Theia run `che:theia init` in `/tmp/theia` directory or use `'2. Init che:theia'` command.
+
+```
+[che-dev]
+$ cd /tmp/theia
+$ che:theia init
+```
+
+### Step 3: Build
+
+The `'3. Build che:theia'` command builds Che-Theia. It runs `yarn` in `/tmp/theia` directory.
+
+```
+[che-dev]
+$ cd /tmp/theia
+$ yarn
+```
+
+### Step 4: Launch
+
+Before launching Che-Theia we have to prepare dedicated directories for plugins, default plugins and for projects, which is used as workspace directory.
+
+Use `'4.1. Prepare theia-* dirs'` command to prepare all the directories in one click. The command will run the following in theia-editor container.
```

Comment:
`theia-editor` component
but container is `theia-ide`
isn't it?
![image](https://user-images.githubusercontent.com/1636395/74955399-46f52380-540d-11ea-8f0a-36b624a70441.png)

Reply:
It looks like it's a bug. If the component has an alias, it's container name must be equal to this alias. At least from the user point of view it's logical.

Reply:
In fact, the container on your screenshot  named as `theia-idebqq`.